src/hazelcore/Operators_Exp.re

```diff
@@ -55,3 +55,34 @@ let is_Space =
   fun
   | Space => true
   | _ => false;
+
+let precedence =
```

Comment:
you should reference the OCaml syntax definition to make sure we are giving these things the right precedence. I don't think Comma, Cons and Space should all be 0 for example...

Reply:
Ok. I was confused about these 3 operators, I'll take a look at what OCaml does.

Reply:
OK, you can also look at the current skel expr parser. the order that the operator declarations appears is the precedence order.

Reply:
Oh ok that's very helpful.